src/components/Order-card/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { Card, CardContent, Divider, Grid } from '@mui/material'
+import OrderPrice from './OrderPrice'
+import { Typography } from '../Typography'
+import OrderList from './OrderList'
+import { BasketItemsResponse } from '../../types'
+import { SUM, TARIFF, DELIVERY } from '../../pages/order-page/constants'
+import { cardStyle, cardContentStyle, dividerStyle } from './style'
+
+type Props = {
+  orderList: BasketItemsResponse[] | null
+}
+
+const OrderCard: React.FC<Props> = ({ orderList }) => {
+  const totalPrice = `${orderList?.reduce((total, order) => total + order.quantity * order.product.prices[0].price, 0)}₴`
```

Comment:
use [Intl.NumberFormat](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) here for formatting